docs/Tor.md

```diff
@@ -140,3 +140,26 @@ eclair.socks5.randomize-credentials = true
 features from using Tor, use both.
 
 Note, that bitcoind should be configured to use Tor as well (https://en.bitcoin.it/wiki/Setting_up_a_Tor_hidden_service).
+
+### Blockchain watchdogs
+
+Eclair version 0.5.0 introduced blockchain watchdogs, that fetch bitcoin headers from various sources in 
+order to detect whether the node is being eclipsed. Eclair supports four sources atm:
+
+* blockchainheaders.net
+* blockcypher.com
+* blockstream.info
+* mempool.space
+
+Once `eclair.socks5.enabled` is set to `true` blockchain watchdogs connect to their respective sources over Tor.
+
+The most Tor-friendly sources are `blockstream.info` and `mempool.space` since they have native onion endpoints for their APIs.
+
+Tor support for `blockchainheaders.net` is not implemented (yet), so it gets automatically disabled when `eclair.socks5.enabled = true` to protect user's privacy.
+
+`blockcypher.com` can be flaky when used over Tor. It imposes rate limits and sometimes (rather often in fact) requires solving CAPTCHA.
+If you experience similar troubles with `blockcypher.com` use this config parameter to disable it:
+
+```
+eclair.blockchain-watchdog.disabled-backends = ["blockcypher.com"]
+```
```

Comment:
I think it would be even more useful if we exposed an `enabled-backends` list instead.
Let's put all supported watchdogs in the default configuration, and users can override this to use whatever subset they like.